__tests__/utils.tsx

```diff
@@ -137,23 +137,22 @@ describe('getPathInfo()', () => {
 
     test('contains right query when path is an uuid', async () => {
       const apiRequest = await getApiRequest(LanguageCode.En, '/12345')
+      const apiBody = (await apiRequest.json()) as { variables: unknown }
 
-      expect(await apiRequest.text()).toEqual(
-        expect.stringContaining('uuid(id: 12345)')
-      )
+      expect(apiBody.variables).toEqual({ id: 12345, alias: null })
     })
 
     describe('contains right query when path is an uuid', () => {
       test.each([LanguageCode.En, LanguageCode.De])(
         'lang = %p',
         async (lang) => {
           const apiRequest = await getApiRequest(lang, '/path')
+          const apiBody = (await apiRequest.json()) as { variables: unknown }
 
-          expect(await apiRequest.text()).toEqual(
-            expect.stringContaining(
-              `uuid(alias: { instance: ${lang}, path: \\"/path\\" })`
-            )
-          )
+          expect(apiBody.variables).toEqual({
+            alias: { instance: lang, path: '/path' },
+            id: null,
+          })
```

Comment:
That looks like an implementation detail to me. I'd probably test that on a higher-level while mocking the request with something like msw.